qbft/state.go

```diff
@@ -28,22 +24,22 @@ type IConfig interface {
 	GetNetwork() Network
 	// GetTimer returns round timer
 	GetTimer() Timer
+	// GetSignatureVerifier returns the signature verifier for operator signatures
+	GetSignatureVerifier() types.SignatureVerifier
+	// GetRoundCutOff returns the round cut off
```

Comment:
```suggestion
	// GetCutOffRound returns the round cut-off
```

Reply:
According Webster, cutoff should be one word when used as noun or adjective

Reply:
@GalRogozinski Sorry, I trusted my IDE. Just aligning the comment to the function name is fine